src/parallel.cpp

```diff
@@ -564,11 +565,12 @@ void HandleBlockMessageThread(CNode *pfrom, const string &strCommand, const CBlo
         int nTotalThinBlocksInFlight = 0;
         {
             LOCK2(cs_vNodes, pfrom->cs_mapthinblocksinflight);
+
             // Erase this thinblock from the tracking map now that we're done with it.
-            if (pfrom->mapThinBlocksInFlight.erase(inv.hash))
```

Comment:
erase vs count ... we don't want to "Erase this thinblock from the tracking map" anymore?